### DISCUSSION TITLE
[2024 PROJECTS TURN-IN] Final Projects discussions and links

### DISCUSSION THREAD

**Discussion by williamgilpin** [williamgilpin/cphy#82] [projects]
Please make one post per group or project. Please make sure that your repository is publicly viewable.

**Comment by carsonmcvay17** [comment]
This is the project repo for group B (Akash, Bishoy, Carson, and Leslie)
https://github.com/carsonmcvay17/class_project.git

**Reply by williamgilpin** [reply]
congrats on being first :)

**Comment by psdalwadi** [comment]
Group A Project repo (Ali, Cesar, Marcus, Pavan, Peter)
https://github.com/alibokhamseen/error_twirling

**Reply by williamgilpin** [reply]
thank you!

**Comment by SomewhatJustin** [comment]
Group D Project repo https://github.com/SomewhatJustin/comp-phy-project
Group members: Max Demarr, Brendan Oloughlin, Samuel "Justin" Fowler

**Reply by williamgilpin** [reply]
thank you!

**Comment by svachher1** [comment]
PHY 381C
Group D Project repo https://github.com/svachher1/ase
Group Members: Sahil Vachher, Claiborne Jones

**Reply by williamgilpin** [reply]
thanks!

**Comment by amalbumbia** [comment]
PHY 329
Group F Project: https://github.com/r-siddiqi/Hofstadter/tree/main
Members: Amal Bumbia, Rida Siddiqi, Maddox (Max) Wroblewski

**Reply by williamgilpin** [reply]
this looks great! amazing gifs

**Comment by milestesta** [comment]
PHY 381C
Group E Project: https://github.com/milestesta/RMT_Tokamak
Members Joe Roll, Kieran McDonald, and Miles Testa

**Comment by jicchen** [comment]
PHY 381C
Group C final project: https://github.com/Johnny880724/AFSA_material
Members: Fu-Xiang Chen, Rui-An Chang, Johnny Tsao, Jie-Cheng Chen

**Comment by hasithv** [comment]
PHY 381C
Group F project: [https://github.com/devddesai/metrobike](https://github.com/devddesai/metrobike)
Members: Hasith Vattikuti, Eric Liang, Dev Desai, Viren Govin

**Comment by joeyschmidt97** [comment]
PHY 381C
Group H project: [https://github.com/joeyschmidt97/ND_path_finder](https://github.com/joeyschmidt97/ND_path_finder)
Members: Joseph Schmidt, Subin Lee, Octavio Ivan Guerrero Ramirez, Michael Thumann

**Comment by Nachiket-Yadav** [comment]
PHY 329
Group B Teaching project: https://github.com/Nachiket-Yadav/CPhysics_PINN_Teaching/tree/main
Members: Christian Gonzalez, Nachiket Yadav, Om Gawankar, Sanju Addanki

**Comment by jreising3** [comment]
Group I:  https://github.com/jreising3/cphys_project
(Parth Dave, Jon Reising, and Armin)

**Comment by MagfluxP** [comment]
This is Group G's final project
https://github.com/LemonadeLiu/Group-G-Project
(Yichen Guo, Xiaohui Liu, Xinyue Peng)

**Comment by thomasleishner** [comment]
PHY 329
Group G Project: https://github.com/thomasleishner/Ising_Model_Project
Members: Thida Lay-Sok, Thomas Leishner, Connie Yang

**Comment by dennisrhom** [comment]
PHY 329 
Group E Project: https://github.com/Polkadoty/scikit-aero
Members: Dennis Hom (drh3449) , Andrew Doty (ajd3982)

**Reply by Polkadoty** [reply]
Thanks Dennis, was happy to work on this with you :)

**Comment by s3a2** [comment]
PHY 329
Group H Project: https://github.com/meatsupplies/IsingModel
Members: Shaan Sidhwani, Blake Steadham, Anthony Hoang, Filippo Meoni

**Comment by mehtan-rahman** [comment]
PHY 381C
Group A Project: [https://github.com/mehtan-rahman/PyVicsek](https://github.com/mehtan-rahman/PyVicsek)
Members: Mehtan Rahman, Sharon Siby

**Comment by MrLou1976** [comment]
PHY 329
Group C Project: https://github.com/MrLou1976/PHY_329_Final_Project
Members: Pratham Gujar, Caitlin Coder, Louis Sauceda
